### DISCUSSION TITLE
Update to 4.6.2 - Front End did not changed

### DISCUSSION THREAD

**Discussion by JN-CSIRT** [DependencyTrack/dependency-track#2099] [Q&A] [answered]
Hello.
normally the DT update is done by pull new docker images (apiserver and frontend), recreate container and start DT by docker-compose again. Right?
After we did it, we see at the FrontEnd no new features mentioned in the documentation.
E.g.:
1) no recurring task configuration as in [recurring tasks](https://docs.dependencytrack.org/getting-started/recurring-tasks/)
2) templates still not be editable.

The DT version in the footer's right corner shows the version 4.6.2

Did we forget something?

**Comment by stevespringett** [comment]
https://docs.dependencytrack.org/changelog/

**Comment by JN-CSIRT** [comment, marked as answer]
Hi Steve,
thank you for Link - I seen it already. 
Btw. it is solved somehow - after 30 minutes after one more site "hard" reload - I can see everything new!

solved!
